lib/ListActions/ListActions.js

```diff
@@ -648,8 +648,17 @@ var ListActions = module.exports = kind(
 	* @private
 	*/
 	shouldStack: function() {
+		var i = 0,
+		    nVisibleComps = 0;
+
+		for (i = 0; i<this.listActionComponents.length; i++) {
+			if (this.listActionComponents[i].getShowing()) {
+				nVisibleComps++;
```

Comment:
Thanks for your comment. I replaced conventional array iterator with assigning member in iterating condition block like you mentioned.